dist/js/multiselect.js

```diff
@@ -1,755 +1,1812 @@
-/*

```

Comment:
Because of the whitespace difference this will be hard to comment on...we'll see